web/src/js/__tests__/components/EventLogSpec.js

```diff
@@ -0,0 +1,57 @@
+jest.mock('../../components/EventLog/EventList')
+
+import React from 'react'
+import renderer from 'react-test-renderer'
+import TestUtils from 'react-dom/test-utils'
+import ConnectedComponent, { EventLog } from '../../components/EventLog'
```

Comment:
Maybe call this EventLog / PureEventLog? I was reading the code below and it was not really clear to me what `ConnectedComponent` is.